.env.sample

```
@@ -0,0 +1,13 @@
+DATABASE_URL = "mysql://db_username:password@db_host:db_port/database_name"
```

```suggestion
DATABASE_URL = "rdbms://db_username:password@db_host:db_port/database_name"
```
다른것과 마찬가지로 rdbms 또한 중의적으로 표현해주세요~